ts/features/itwallet/issuance/screens/ItwIssuanceCredentialFailureScreen.tsx

```diff
@@ -120,6 +120,22 @@ const ContentView = ({ failure }: ContentViewProps) => {
             )
           )
       }
+    },
+    ASYNC_ISSUANCE: {
+      title: I18n.t("features.itWallet.issuance.asyncCredentialError.title"),
+      subtitle: I18n.t("features.itWallet.issuance.asyncCredentialError.body"),
+      pictogram: "pending",
+      action: {
+        label: I18n.t(
+          "features.itWallet.issuance.asyncCredentialError.primaryAction"
+        ),
+        onPress: () =>
+          closeIssuance(
+            I18n.t(
+              "features.itWallet.issuance.asyncCredentialError.primaryAction"
+            )
+          )
+      }
```

Comment:
According to the [FIGMA](https://www.figma.com/design/pVD9XCba7crkg5yOIdJCXl/%5BIT-Wallet%5D---%E2%9C%85-Approved-only-%E2%9C%85?node-id=1084-33991), we should navigate to the messages section

Reply:
Right. I parameterized the `close` action to specify a screen to navigate to in 1238638c3942e34ddee39b5bbea905fcd88b88d1.